modules/entity_version_history/tests/src/FunctionalJavascript/HistoryOverviewTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\entity_version_history\Functional;
+
+use Drupal\entity_version_history_test\EventSubscriber\TestHistoryOverviewAlterEventSubscriber;
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\node\Entity\Node;
+
+/**
+ * Tests the history overview table.
+ *
+ * @group entity_version_history
+ */
+class HistoryOverviewTest extends WebDriverTestBase {
```

Comment:
Can't this be a kenrnel test in whiuch you load the controller (like the routing system does) and you check it's buid method? You are already testing in the other test the fact that we have a History path and local task so it is no longer really required.

Reply:
I think it's good to test what we get rendered in in the markup and we have those elements in place with the correct order.

Reply:
1. You can test the rendered markup in Kernel.
2. You are testing the URL parts (or can test what is missing) in the other test no?

Reply:
We will leave the functional test in place.